test/NuGet.Core.Tests/NuGet.Protocol.Tests/Resources/RepositorySignatureResourceTests.cs

```diff
@@ -0,0 +1,111 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+using Moq;
+using Newtonsoft.Json.Linq;
+using NuGet.Packaging.Core;
+using NuGet.Protocol.Core.Types;
+using Test.Utility;
+using Xunit;
+
+namespace NuGet.Protocol.Tests
+{
+    public class RepositorySignatureResourceTests
+    {
+        [Fact]
+        public async Task RepositorySignatureResource_Basic()
+        {
+            // Arrange
+            var source = $"http://unit.test{Guid.NewGuid()}/v3-with-flat-container/index.json";
```

Comment:
Not sure why you need randomness in the URL, but if so, put `/` before `{Guid.NewGuid()}`

    http://unit.test/{Guid.NewGuid()}/v3-with-flat-container/index.json

.test should be the root domain, not .test1 or .testGUID.

Also update subsequent tests ([here](https://github.com/NuGet/NuGet.Client/pull/2083/files#diff-9bcefcb5c7f544fb506b5d5e916c4ee5R44), [here](https://github.com/NuGet/NuGet.Client/pull/2083/files#diff-9bcefcb5c7f544fb506b5d5e916c4ee5R61), and [here](https://github.com/NuGet/NuGet.Client/pull/2083/files#diff-9bcefcb5c7f544fb506b5d5e916c4ee5R75)).

Reply:
that's because of cache, we have some tests which are using same url.